services/pub/pub-downloads.service.js

```diff
@@ -0,0 +1,54 @@
+import Joi from 'joi'
+import { BaseJsonService, pathParams } from '../index.js'
+import { metric } from '../text-formatters.js'
+import { nonNegativeInteger } from '../validators.js'
+import { baseDescription } from './pub-common.js'
+
+const description = `${baseDescription}
+  <p>This badge shows a measure of how many developers have downloaded a package. This provides a raw measure of the overall sentiment of a package from peer developers.</p>`
```

Comment:
The naming of `downloadCount30Days` seems to imply it's a download count during the last month, would it be helpful to indicate that in the first sentence?

The second sentence doesn't feel that helpful to me, do you feel strongly about keeping it?

Reply:
Yes, Updated the first one and removed the second.